### ISSUE TITLE
Support rpm-ostree package layering

### ISSUE THREAD

**Issue by jlebon** [rhinstaller/anaconda#799]
Now that rpm-ostree supports [package layering](http://rpm-ostree.readthedocs.io/en/latest/manual/administrator-handbook/#package-layering), it would be beneficial to add support for this at kickstart time, e.g. for packages that are needed right from first boot.

Related downstream RHBZ: https://bugzilla.redhat.com/show_bug.cgi?id=1209256

**Comment by rvykydal** [comment]
Do you care to make this happen in RHEL 7? If so I'd recommend to file a downstream RFE for anaconda. Are there other cases that could benefit from the feature in installer? We'd like to have some grounds for prioritizing the requrest.

**Comment by jlebon** [comment]
We would probably want to target Fedora first. Should I still make an RFE on RHBZ for that?

> Are there other cases that could benefit from the feature in installer? We'd like to have some grounds for prioritizing the requrest.

Anywhere where additional packages are required to boot and connect to the node (i.e. kernel modules). A prerequisite for this though is to make `rpm-ostree` regenerate the initramfs when layering such packages.

Note that there is also a similar card on the Platform Infrastructure board for system containers: https://trello.com/c/fy90Biw2. I think the focus for that one is more for guest agents, though I think there still needs to be a discussion to clarify when to use each method.

**Comment by jkonecny12** [comment]
> We would probably want to target Fedora first. Should I still make an RFE on RHBZ for that?

Yes please, it's harder for us to follow GitHub issues here and BZ at one time.

**Comment by imcleod** [comment]
@jlebon - Did we end up doing an RFE for this in RHBZ?  And if not, can we?

**Comment by jlebon** [comment]
OK, I opened https://bugzilla.redhat.com/show_bug.cgi?id=1434912.

**Comment by poncovka** [comment]
Thank you for reporting this issue! The issue tracker will be closed on 3 June 2019. Please make sure that your issue is also reported at https://bugzilla.redhat.com/. We are handling a high number of issues and many of them involve other components. Bugzilla allows us to better process your issues and collaborate on them with other teams. We are sorry for the inconvenience.
